tutorial/source/bayesian_regression.ipynb

```diff
@@ -207,7 +207,7 @@
    "source": [
     "### Model\n",
     "\n",
-    "We now have all the ingredients needed to specify our model. First we define priors over $w$ and $b$.  Because we're uncertain about the parameters a priori, we'll use relatively wide priors $\\mathcal{N}(\\mu = 0, \\sigma = 10)$.  Then we wrap `regression_model` with `random_module` and sample an instance of the regressor, `lifted_rm`. We then run the regressor forward on the inputs `x_data`. Finally we use the `pyro.observe` statement to condition on the observed data `y_data`. Note that we use the same fixed observation "
+    "We now have all the ingredients needed to specify our model. First we define priors over $w$ and $b$.  Because we're uncertain about the parameters a priori, we'll use relatively wide priors $\\mathcal{N}(\\mu = 0, \\sigma = 10)$.  Then we wrap `regression_model` with `random_module` and sample an instance of the regressor, `lifted_rm`. We then run the regressor forward on the inputs `x_data`. Finally we use the `pyro.observe` statement to condition on the observed data `y_data`. Note that we use the same fixed observation noise that was used to generate the data."
```

Comment:
can you change `lifted_rm` to `lifted_reg_model` (as it is in the code?)
there are 2 other places where `sampled_rm` is incorrectly named in the last code snippet. can you fix them as well?